app/classes/site_data.rb

```diff
@@ -142,6 +138,34 @@ class SiteData
     contributing_users: "contribution > 0"
   }.freeze
 
+  # Non-default unified queries for stats for the entire site
+  # rubocop:disable Layout/MultilineMethodCallIndentation
+  # Rubocop 1.30.0 wants to allgn "where" with the open brace on the next line.
+  FIELD_QUERIES = {
+    contributing_users:
+      User.
+        where(User[:contribution].gt(0)),
+    observations_with_voucher:
+      Observation.
+        where(specimen: true).
+        where("LENGTH(observations.notes) >= 10").
```

Comment:
This could also be Arel-ized. I left it as-is because `
- LENGTH` seems to be a basic (universal?) SQL function
- the replace Arel looked complicated to me. I.e., something like:
```ruby
Arel::Nodes::NamedFunction.new('LENGTH', [Observation.arel_table[:notes]]).gteq(10)
```
But I wouldn't mind Arel if there's a shorter way to it, or even if others prefer the longer Arel to SQL.

Reply:
Can you help me understand these named functions?  Are they really database-agnostic?  Couldn't we name any random mysql-specific function here?   The point, after all, is not to turn things into Arel, but to make it database-independent.  But there must be an Arel string length method along the lines of "gt", etc., right?  As you say, this must be a universal.  But I wouldn't be completely surprised if it's not spelled "LENGTH" in every database.

Reply:
@JoeCohen — Congrats on the Arel! Here's a [reference for arel-extensions](https://github.com/Faveod/arel-extensions). 

I believe @pellaea is right about named functions, they do not create database-agnostic SQL but rather allow us to write db-specific SQL, which is not what we're going for. It took me awhile to figure that out.  

But arel-extensions is here to help. What I think you'd want is:
```ruby
Observation.
  where(specimen: true).
  where(Observation[:notes].length >= 10)
```
Note that it allows you to also keep the `>=` operators. It's pretty convenient!